all-in-one/docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  storage-node:
+    image: storjlabs/storage-node:${VERSION}
+    environment:
+    - SATELLITE_ADDR=satellite:7777
+    links:
+    - satellite
+  satellite:
```

Comment:
I had to add the following to get around an issue related to the satellite not being able to find certification keys in `/root/.storj/hc`
```
volumes:
- ~/.storj/captplanet:/root/.storj
```

Reply:
*
- ~/.storj/capt:/root/.storj

Reply:
I think this is because you pulled an old version of the images. `VERSION=latest` won't work until after this PR gets merged.